README.md

```diff
@@ -1,60 +1,67 @@
 # judge-prolog
 
-Currently a PLUnit tester only
+A judge for the Prolog language.
 
-Can be tested with:
+## Format of the QuickCheck tests
 
-```bash
-docker run --rm  -v $(realpath .):/home/runner/workdir dodona-prolog ./test/test.bash
-```
-
-After a build using
-
-```bash
-docker build -t dodona-prolog .
-```
-
-
-The `main.sh` file is form the [docker-images](https://github.ugent.be/dodona/docker-images) repo, it should not be edited and will be removed.
+QuickCheck testfiles should be named with a name ending in `.qc.pl`. The name
+before the extension will be used as tabname.
 
+All tests must have a name starting with `prop_`. There should be at least one
+blank line separating the definition from other things. A property must not
+contain empty lines in its definition (you may use comments to fill the space).
+Tests may contain comments. Comments within the test will be shown to the
+student, comments outside it wil not.
 
-## Todos
-
-- [x] Memory management
-- [ ] Using the partial format
-- [x] Add quickcheck (see the `feature/quickcheck` branch and PR)
-- [ ] make text below clearer
-
-## Format of the Quickcheck tests
+The name of the QuickCheck predicates should start with `prop_`
 
-All tests must have a name starting with `prop_`. There should be at least one blank line sparating the definition from other thing. Tests may contain comments. Comments within the test will be shown to the student, comments outside it wil not.
+```prolog
+prop_reverse_twice(L:list(integer)) :-
+    % This comment is shown to students
+    my_reverse(L, R),
+    my_reverse(R, L).
+```
 
 ## Format of PLUnit testfiles
 
-The file will be splitted up in parts between the tags `:- begin_tests(...).` and `:- end_tests(...).`. All the lines specified between the parst above the current part wil be used.
+PLUnit testfiles should be named with a name ending in `.unit.pl`. The name
+before the extension will be used as tabname.
+
+The testfile will be splitted up in parts between the tags
+`:- begin_tests(...).` and `:- end_tests(...).`. All the lines specified before
+the `begin_tests` and not in an other part can be used.
```

Comment:
```suggestion
the `begin_tests` and not in another part can be used.
```

Reply:
Also not sure what you mean by, "can be used". I would assume it would say "will not be used".